README.md

```diff
@@ -75,6 +75,16 @@ Clone the repository and execute:
 npm test
 ```
 
+### Debugging
+
+install the following package in VS Code
+
+```
+DmitriyMuraviov.vscode-jasmine-test-runner
```

Comment:
since you are adding vscode specific config files, you can also include an extension.json file with this recommendation

Reply:
I'll see how I can do that